client/components/events/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from "./EventList";
```

Comment:
I wonder if it'd be better to make these exports explicit so we know exactly what we're exporting